s3parq/publish_parq.py

```diff
@@ -45,14 +45,14 @@ def check_partitions(partitions: iter, dataframe: pd.DataFrame)->None:
     logger.debug("Done checking partitions.")
 
 def check_redshift_params(redshift_params: dict):
-    expected_params = ["schema_name", "table_name", "iam_role", "region", "cluster_id", "host", "port", "db_name"]
+    expected_params = ["schema_name", "table_name", "iam_role", "region", "cluster_id", "host", "port", "db_name", "ec2_user"]
     logger.debug("Checking redshift params are correctly formatted")
-    number_redshift_params = 8
+    number_redshift_params = 9
```

Comment:
you can remove this and change the if statement to be `if len(redshift_params) != len(expected_params):`

Reply:
good idea!